data/gegno/gegno ships.txt

```diff
@@ -0,0 +1,1199 @@
+
+ship "Conglomerate"
+	sprite "ship/gegno conglomerate"
+	thumbnail "thumbnail/gegno conglomerate"
+	attributes
+		category "Heavy Freighter"
+		licenses
+			"Gegno Civilian"
+		"cost" 1692000
+		"shields" 0
+		"hull" 13300
+		"required crew" 32
+		"bunks" 68
+		"mass" 760
+		"drag" 8.9
+		"heat dissipation" 0.7
+		"fuel capacity" 400
+		"cargo space" 415
+		"outfit space" 200
+		"weapon capacity" 0
+		"engine capacity" 136
+		weapon
+			"blast radius" 124
+			"shield damage" 960
+			"hull damage" 1160
+			"hit force" 1650
+	outfits
+		"C7 Brawl Cell"
+		"R02 Battlezone Battery"
+		
+		"RG18 Torch Thruster"
+		"RG18 Torch Steering"
+		"RG15 Torch Steering"
+		"Hyperdrive"
+	engine -34.5 220
+	engine 34.5 220
+	leak "leak" 30 40
+	leak "flame" 80 20
+	explode "huge explosion" 10
+	explode "large explosion" 30
+	explode "medium explosion" 38
+	explode "small explosion" 26
+	"final explode" "final explosion large"
+	description `Over time, the Gegno needed larger vessels to carry materials to and from new worlds - ones that would be easier to build and operate than their older, outdated generation ships, but larger than their shuttlecraft. The Conglomerate was designed to fill that exact role.`
+
+ship "Gypsum"
+	sprite "ship/gegno gypsum"
+	thumbnail "thumbnail/gegno gypsum"
+	attributes
+		category "Light Freighter"
+		licenses
+			"Gegno Civilian"
+		"cost" 1692000
+		"shields" 0
+		"hull" 6000
+		"required crew" 5
+		"bunks" 15
+		"mass" 380
+		"drag" 3
+		"heat dissipation" 0.8
+		"fuel capacity" 500
+		"cargo space" 220
+		"outfit space" 240
+		"weapon capacity" 42
+		"engine capacity" 120
+		"drill port" 1
+		"drill lock" -1
+		weapon
+			"blast radius" 72
+			"shield damage" 320
+			"hull damage" 620
+			"hit force" 720
+	outfits
+		"Excavator"
+		
+		"C7 Brawl Cell"
+		"R01 Skirmish Battery"
+		
+		"RG18 Torch Thruster"
+		"RG15 Torch Steering"
+		"RG18 Torch Steering"
+		"Hyperdrive"
+	engine 0 124
+	gun 0 -117 Excavator
+	leak "leak" 60 50
+	leak "flame" 60 80
+	explode "large explosion" 14
+	explode "medium explosion" 24
+	explode "small explosion" 26
+	explode "tiny explosion" 10
+	"final explode" "final explosion medium"
+	description `To fuel their increasing need for materials, the Gegno designed a new ship class for the sole purpose of mining. The resulting platform, the Gypsum, is not much more than a flying drill with cargo containers, engines, and a bridge attached behind it, making it cost-efficient and easy to manufacture.`
+	description `	Because of the Gegno's limited technology, the Gypsum is much more sluggish than similarly sized ships elsewhere in the galaxy, making it too slow to chase faster asteroids. For those asteroids it can catch up to, the powerful Excavator Drill is able to grind materials out with ease.`
```

Comment:
@Zitchas It's inferior because it *can't* go after fast asteroids in-game.

Reply:
```suggestion
	description `	Because Gegno engines are not very powerful, the Gypsum has difficulty chasing faster moving asteroids. In order to alleviate this problem without needing to massively increase the engine space on the ship, the Gegno have instead developed the powerful Excavator Drill, which can shreds asteroids in a matter of seconds, meaning the Gypsum only needs to get near an asteroid for a mere moment in order to mine it.`
```